misc/python/materialize/mzbuild.py

```diff
@@ -223,6 +233,13 @@ def __init__(self, rd: RepositoryDetails, path: Path, config: Dict[str, Any]):
         self.strip = config.pop("strip", True)
         self.extract = config.pop("extract", {})
         self.rustflags = config.pop("rustflags", "")
+        if rd.coverage:
+            self.rustflags += " -Zinstrument-coverage"
```

Comment:
I think ```-C link-dead-code``` is also needed here. Otherwise one of the tools involved in generating the final report borks (I must admit I do not recall which one). I think it was ```llvm-cov report```